src/components/SignerPanel/SignerPanel.js

```diff
@@ -108,10 +141,23 @@ class SignerPanel extends React.Component {
     })
   }
 
+  async signMessage(message, account) {
+    const { walletWeb3 } = this.props
+    return new Promise((resolve, reject) => {
+      walletWeb3.eth.personal.sign(message, account, (err, res) => {
```

Comment:
Do we actually need to instantiate a new promise here? 

I understand that this is repeating the pattern in `signTransaction`, but perhaps we could just return the promise from `walletWeb3.eth.personal.sign`